pkg/systemstatsmonitor/system_stats_monitor.go

```diff
@@ -89,6 +90,9 @@ func NewSystemStatsMonitorOrDie(configPath string) types.Monitor {
 		ssm.memoryCollector = NewMemoryCollectorOrDie(&ssm.config.MemoryConfig)
 	}
 	if len(ssm.config.OsFeatureConfig.MetricsConfigs) > 0 {
+		// update the known_modules_path to relative the system-stats-monitors path
+		ssm.config.OsFeatureConfig.KnownModulesConfigPath = filepath.Join(filepath.Dir(configPath),
```

Comment:
We should only do this when `configPath` is relative, and we should update the README.md about this https://github.com/kubernetes/node-problem-detector/blob/master/pkg/systemstatsmonitor/README.md#os-features

Reply:
updated the readme. 
>We should only do this when configPath is relative 
Could you please clarify.

Reply:
the suggestion here is to do `filepath.Join` only when `KnownModulesConfigPath` is a relative path. So it will work fine with this fix: https://github.com/kubernetes/kubernetes/pull/101629